Assets/MRTK/Core/Utilities/WindowsDevicePortal/DevicePortal.cs

```diff
@@ -745,13 +777,10 @@ public static async Task<NetworkInterfaces> GetWiFiNetworkInterfacesAsync(Device
         /// <returns>The finalized URL with http/https prefix.</returns>
         public static string FinalizeUrl(string targetUrl)
         {
-            string ssl = Rest.UseSSL ? "s" : string.Empty;
+            string ssl = DevicePortal.UseSSL ? "s" : string.Empty;
 
             if (targetUrl.Contains(DeviceInfo.LocalMachine) || targetUrl.Contains(DeviceInfo.LocalIPAddress))
-            {
-                targetUrl = $"{DeviceInfo.LocalIPAddress}:10080";
-                ssl = string.Empty;
-            }
+                targetUrl = UseSSL ? $"{DeviceInfo.LocalIPAddress}:10443" : $"{DeviceInfo.LocalIPAddress}:10080";
```

Comment:
Nit: Add back the brackets i.e.
if (...)
   ...

becomes:

if (...)
{
   ...
}

Even though there's only one line here, the preference in this codebase is to have explicit scopes. This is to avoid a class of bugs where you add another line and it doesn't end up actually falling under the if () statement as you expect. So yeah this is a tradeoff of verbosity vs. protection against a class of bugs.

Reply:
Done